src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.cc

```diff
@@ -21,7 +21,7 @@
 #include <grpc/support/alloc.h>
 #include <grpc/support/log.h>
 #include <grpc/support/string_util.h>
-#include <openssl/ssl.h>
```

Comment:
The tests keep failing because they can't find `ssl.h` but it builds just fine on my local. @ZhenLian